src/main/java/controllers/service/ArticleService.java

```diff
@@ -0,0 +1,32 @@
+package controllers.service;
+
+import controllers.service.interfaces.ArticleServiceInterface;
```

Comment:
I would prefer IArticleService name and in the servise package

Reply:
Ok